README.md

```diff
@@ -1476,7 +1476,7 @@ OPTIONS:
         --value=VALUE                extended value for create, update, list filter
         --property=VALUE             name of property to set
         --id=VALUE                   resource identifier (modify,delete,show)
-        --config-file=VALUE          read parameters from file in YAML format, current=/Users/dwosk/.aspera/ascli/config.yaml
+        --config-file=VALUE          read parameters from file in YAML format, current=/Users/laurent/.aspera/ascli/config.yaml
```

Comment:
```suggestion
        --config-file=VALUE          read parameters from file in YAML format, current=~/.aspera/ascli/config.yaml
```
I've seen these usernames change back and forth a few times.. Could change the doc to a more generic/platform-independent name (like `<username>`) or simply `~`?

Reply:
Yes, I have seen as well, I have fixed this in a next version, I can push.